misc/Kconfig

```diff
@@ -165,6 +172,40 @@ config NO_OPTIMIZATIONS
 
 endchoice
 
+choice
+	prompt "C++ Standard"
+	default STD_CPP11
+	help
+	  C++ Standards.
+
+config STD_CPP98
+	bool "C++ 98"
+	help
+	  1998 C++ standard as modified by the 2003 technical corrigendum 
```

Comment:
Pedantry: but I don't think GCC with -std=c++98 promises to implement c++03, does it?  All but certain those are different standard levels to the compiler.

Reply:
The list of C++ standard supported by GCC comes from this page : [C++ Standards Support in GCC](https://www.gnu.org/software/gcc/projects/cxx-status.html)